spec/requests/api/groups_spec.rb

```diff
@@ -3,8 +3,9 @@
 describe API::API, api: true  do
   include ApiHelpers
 
-  let(:user1) { create(:user) }
+  let(:user1) { create(:user, can_create_group: false) }
   let(:user2) { create(:user) }
+  let(:user3) { create(:user) }
```

Comment:
We need a user that is not member of a group. Both user1 and user2 are set to be members of on of the groups.
I’ve send this PR one month ago, I really don’t remember all the details now…